docs/conf.py

```diff
@@ -305,6 +308,38 @@ def add_google_calendar_secrets(app, docname, source):
     if docname == 'community/meeting_schedule':
         source[0] = source[0].replace('{API_KEY}', GOOGLE_CALENDAR_API_KEY)
 
+class FilterSphinxWarnings(logging.Filter):
```

Comment:
I thought you could filter warnings like this: https://github.com/sphinx-gallery/sphinx-gallery/pull/521/files

But I can't yet find documentation on this.

Reply:
Usually yes - but this warning in particular you can't (or I couldn't!). See https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-suppress_warnings for a list of warnings that can be explicitly silenced. I think it falls into a similar category as https://github.com/sphinx-doc/sphinx/issues/9750 - if you find a way to filter it, that would be great!

Reply:
Oh man I didn't know any of this. Using `warnings.filterwarnings` in the `conf.py` file - how does this even work? I can't find documentation of this. Is this a Sphinx thing?

I guess in https://github.com/sphinx-gallery/sphinx-gallery/pull/521/ , the warning is passed on by the extension Sphinx-Gallery? In the Sphinx docs it says:

> Then extensions can also define their own warning types.

Does that mean that extension warning types are also able to be suppressed?

Anyway, I had no idea, this seems reasonable way to fix it AFAICT! Thanks!